### PR TITLE
docs: Add issue template

### PR DESCRIPTION
**Pull request by tanmay-pnaik**
## Overview

Add issue template

## Notes for reviewer

Adapted from https://github.com/bank-vaults/vault-operator/tree/main/.github/ISSUE_TEMPLATE

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting a bug report!
+
+        Please fill out the template below to make it easier to debug your problem.
+
+        If you are not sure if it is a bug or not, you can contact us via the available [support channels](https://github.com/bank-vaults/bank-vaults.dev/issues/new/choose).
```
[the remark this answers is not in the document]
Comment (fekete-robert) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:12]: https://github.com/bank-vaults/bank-vaults.dev/pull/182/ will restore the support page, so we can update this link and merge this PR after PR#1882

[the remark this answers is not in the document]
Comment (ramizpolic) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:12]: The new link is: https://bank-vaults.dev/docs/support/

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name: 🎉 Feature request
+description: Suggest an idea for the Bank-Vaults Documentation
```
Comment (ramizpolic) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:2]:
```suggestion
description: Suggest an idea for the Bank-Vaults Documentation
labels: [kind/enhancement]
```

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting a feature request!
+
+        Please describe what you would like to change/add and why in detail by filling out the template below.
+
+        If you are not sure if your request fits into Dex, you can contact us via the available [support channels](https://github.com/bank-vaults/vault-operator/issues/new/choose).
```
Comment (ramizpolic) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:11]:
```suggestion
        If you are not sure if your request fits into Dex, you can contact us via the available [support channels](https://bank-vaults.dev/docs/support).
```

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  - type: checkboxes
+    attributes:
+      label: Preflight Checklist
+      description: Please ensure you've completed all of the following.
+      options:
+        - label: I have searched the [issue tracker](https://www.github.com/bank-vaults/vault-operator/issues) for an issue that matches the one I want to file, without success.
```
Comment (ramizpolic) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:17]:
```suggestion
        - label: I have searched the [issue tracker](https://www.github.com/bank-vaults/bank-vaults.dev/issues) for an issue that matches the one I want to file, without success.
```

**Review by ramizpolic — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (tanmay-pnaik) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: Thanks!

File: .github/ISSUE_TEMPLATE/feature_request.yaml (hunk 2 above)
Comment (tanmay-pnaik) [reply .github/ISSUE_TEMPLATE/feature_request.yaml:2]: Fixed!

File: .github/ISSUE_TEMPLATE/feature_request.yaml (hunk 3 above)
Comment (tanmay-pnaik) [reply .github/ISSUE_TEMPLATE/feature_request.yaml]: Fixed!

File: .github/ISSUE_TEMPLATE/feature_request.yaml (hunk 4 above)
Comment (tanmay-pnaik) [reply .github/ISSUE_TEMPLATE/feature_request.yaml]: Fixed!

**Review by ramizpolic — APPROVED** [review]
Much appreciated, @tanmay-pnaik!

**Pull request merged**